DESCRIPTION

```diff
@@ -4,8 +4,8 @@ Author: Dirk Eddelbuettel <edd@debian.org> with contributions
  Mario Frasca, Bryan Lewis, Murray Stokely, Hannes Muehleisen,
  Duncan Murdoch, Jim Hester, Wush Wu, Qiang Kou, Thierry Onkelinx, 
  Michel Lang, Viliam Simko, Kurt Hornik and Radford Neal.
-Version: 0.6.18
```

Comment:
Dev versions get a minor first. Please make it 0.6.18.1

Reply:
I will fix. Anything else apart from this?

Reply:
No, it looks good, and I like it.

I am wondering if we should apply one lesson in 10+ years here and make it vectorised in and out, ie accept a vector of strings and return a vector of ints?  Thoughts?